### PR TITLE
fix: QA fixes for verify screens

### PR DESCRIPTION
**Pull request by bistaastha**
Includes the following fixes:
- Font weight and spacing fixes
- Fix for Loader on Invite expired page
- Adds Toast message on Reset password page
- Adds icon beside error messages

Reference thread for design qa: https://fylein.slack.com/archives/C07PA9SACCR/p1733835884139739

## Summary by CodeRabbit

- **New Features**
	- Enhanced user feedback with new icons for error messages on password fields.
	- Improved button functionality with loading indicators during invite sending.
	- Stricter password validation criteria implemented for better security.

- **Bug Fixes**
	- Consistent text capitalization across various pages for improved user experience.

- **Style**
	- Updated styles for error messages and headers to enhance layout and visual consistency.

- **Tests**
	- Refined test cases for the Reset Password component to ensure accurate user feedback and error handling.

### CODE REVIEW

File: src/app/post-verification/invited-user/invited-user.page.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ export class InvitedUserPage implements OnInit {
     this.isLoading = true;
     this.fg.markAllAsTouched();
     if (this.fg.valid) {
-      from(this.loaderService.showLoader())
+      from(this.loaderService.showLoader('Signing in...', 20000000))
```
Comment (Chethan-Fyle) [inline src/app/post-verification/invited-user/invited-user.page.ts:97]: Why such a big number, @bistaastha ?

**Review by Chethan-Fyle — CHANGES_REQUESTED** [review]
One comment.

File: src/app/post-verification/invited-user/invited-user.page.ts (hunk 1 above)
Comment (bistaastha) [reply src/app/post-verification/invited-user/invited-user.page.ts:97]: was testing a change, will remove this

File: src/app/auth/verify/verify.page.scss [hunk 2]
```diff
@@ -25,10 +25,13 @@
   justify-content: center;
 
   &__verifying-loader {
-    @include loader;
     &__icon {
-      fill: $brand-primary;
+      @include loader;
+      fill: $brand-primary !important;
+      height: 40px;
+      width: 40px;
```
[the remark this answers is not in the document]
Comment (bistaastha) [reply src/app/auth/verify/verify.page.scss:32]: This was a suggested change

**Review by Chethan-Fyle — APPROVED** [review]

**Pull request merged**